openlibrary/catalog/add_book/match.py

```diff
@@ -1,7 +1,7 @@
 import web
 from deprecated import deprecated
+from openlibrary.catalog.utils import expand_record
 from openlibrary.catalog.merge.merge_marc import (
-    build_marc,
     editions_match as threshold_match,
```

Comment:
Remove this comma so psf/black can put this import on a single line. ;-)

```suggestion
    editions_match as threshold_match
```

Reply:
Ahhh fail. Thanks for catching that!